src/__tests__/VoucherRedemption.tsx

```diff
@@ -0,0 +1,263 @@
+        phone_number: '(718) 833-8884',
+      },
+    ],
+    amount_raised: 93500,
+  };
+
+  const voucherResponse: any = {
+    id: 4,
+    seller_id: 'shunfa-bakery',
+    item_type: null,
+    created_at: '2020-07-19T23:15:00.544Z',
+    updated_at: '2020-07-19T23:15:00.544Z',
+    payment_intent_id: 4,
+    refunded: false,
+    purchaser_id: 1,
+    gift_card_detail: {
+      id: 1,
+      gift_card_id: '123asdf',
+      receipt_id: null,
+      expiration: null,
+      created_at: '2020-07-19T23:15:00.582Z',
+      updated_at: '2020-07-19T23:15:00.582Z',
+      item_id: 4,
+      seller_gift_card_id: '123asdf',
+      recipient_id: 1,
+      single_use: false,
+      amount: 2500,
+    },
+  };
+
+  const voucherUpdateResponse: any = {
+    id: 4,
+    seller_id: 'shunfa-bakery',
+    item_type: null,
+    created_at: '2020-07-19T23:15:00.544Z',
+    updated_at: '2020-07-19T23:15:00.544Z',
+    payment_intent_id: 4,
+    refunded: false,
+    purchaser_id: 1,
+    gift_card_detail: {
+      id: 1,
+      gift_card_id:
+        'rphtgeghnxtenpficladyeaklccdkqmxldcshuodqpvaujlshfoxizldomrhpbje',
+      receipt_id: null,
+      expiration: null,
+      created_at: '2020-07-19T23:15:00.582Z',
+      updated_at: '2020-07-19T23:15:00.582Z',
+      item_id: 4,
+      seller_gift_card_id:
+        'crlmuwcorvfwrhnzdfkdjkjhwwitqqwnhhelyahrctehykfustgvrbalipvwiaus',
+      recipient_id: 1,
+      single_use: false,
+      amount: 300,
+    },
+  };
+
+  return {
+    ...jest.requireActual('../utilities/api/interactionManager'),
+    getSeller: jest.fn(() => ({
+      data: shunfaBakeryResponse,
+    })),
+    getVoucher: jest.fn(() => ({
+      data: voucherResponse,
+    })),
+    updateVoucher: jest.fn(() => ({
+      data: voucherUpdateResponse,
+    })),
+  };
+});
+
+beforeAll(() => {
+  axios.defaults.adapter = require('axios/lib/adapters/http');
+});
+
+afterEach(() => cleanup());
+
+const renderVoucher = (providerValues) => (
+  <VoucherProvider>
+    <VoucherPage />
+  </VoucherProvider>
+);
+
+describe('Voucher Page', () => {
+  let providerValues = {};
+  let VoucherProps = {};
+  let VoucherDispatcher = jest.fn();
+  beforeEach(() => {
+    VoucherProps = VoucherTypes.defaultState;
+    providerValues = {
+      VoucherProps,
+      VoucherDispatcher,
+    };
+  });
+
+  test('should display VoucherPage with succesful GET /voucher request', async () => {
+    const { findByText } = renderIntegration(
+      '/voucher/123asdf',
+      renderVoucher(providerValues)
+    );
+
+    const HeaderLabel = await findByText(`Welcome to`, { exact: false });
+    const Header = await findByText(`Shunfa Bakery`, { exact: false });
+    const CardHeader = await findByText('Your available balance');
+    const Amount = await findByText('$25.00');
+    const CodeLabel = await findByText('Voucher Code', { exact: false });
+    const Code = await findByText('123asdf', { exact: false });
+    const NextButton = await findByText(
```

Comment:
Not sure if these should be lower camelcased?

Reply:
yeah ill fix these